examples/hybrid_canister/test/tests.ts

```diff
@@ -0,0 +1,155 @@
+import * as dns from 'node:dns';
+dns.setDefaultResultOrder('ipv4first');
+
+import { getCanisterId, Test } from 'azle/test';
+import { createActor as createActorServer } from './dfx_generated/server';
+import { createActor as createActorServerInitAndPostUpgrade } from './dfx_generated/server_init_and_post_upgrade';
+import { createActor as createActorCanister } from './dfx_generated/canister';
+
+export function getTests(): Test[] {
+    const canisterIdServer = getCanisterId('server');
+    const canisterIdServerInitAndPostUpgrade = getCanisterId(
+        'server_init_and_post_upgrade'
+    );
+    const canisterIdCanister = getCanisterId('canister');
+
+    const originServer = `http://${canisterIdServer}.localhost:8000`;
+    const originServerInitAndPostUpgrade = `http://${canisterIdServerInitAndPostUpgrade}.localhost:8000`;
+    const originCanister = `http://${canisterIdCanister}.localhost:8000`;
+
+    const actorServer = createActorServer(canisterIdServer, {
+        agentOptions: {
+            host: 'http://127.0.0.1:8000'
+        }
+    });
+    const actorServerInitAndPostUpgrade = createActorServerInitAndPostUpgrade(
+        canisterIdServerInitAndPostUpgrade,
+        {
+            agentOptions: {
+                host: 'http://127.0.0.1:8000'
+            }
+        }
+    );
+    const actorCanister = createActorCanister(canisterIdCanister, {
+        agentOptions: {
+            host: 'http://127.0.0.1:8000'
+        }
+    });
+
+    return [
+        {
+            name: 'server',
+            test: async () => {
+                try {
+                    const httpQueryResponse = await fetch(
+                        `${originServer}/http-query`
+                    );
+                    const httpQueryResponseText =
+                        await httpQueryResponse.text();
+
+                    const httpUpdateResponse = await fetch(
+                        `${originServer}/http-update`,
+                        {
+                            method: 'POST'
+                        }
+                    );
+                    const httpUpdateResponseText =
+                        await httpUpdateResponse.text();
+
+                    const candidQueryText = await actorServer.candidQuery();
+                    const candidUpdateText = await actorServer.candidUpdate();
+
+                    return {
+                        Ok:
```

Comment:
These should be broken up into each their own tests, it will be easier to read and easier to figure out where the error is should any of the tests fail

Reply:
The file is very short and there is just one method to test for each, I think that would add unnecessary complexity to the tests.

I don't see how it would be easier to figure out where the error is, the test tell you the exact test that fails, and there are only 3 tests right now

Reply:
Ignore the previous comment, this has been addressed

Reply:
We didn't actually do this for the reasons above